test/e2e/app/app.go

```diff
@@ -686,6 +733,31 @@ func (app *Application) getAppHeight() int64 {
 	return appHeight + 1
 }
 
+func (app *Application) checkBlobHeight(height int64, callsite string) bool {
+	appHeight := app.getAppHeight()
+	if height != appHeight {
+		panic(fmt.Errorf(
+			"got unexpected height in %s request; expected %d, actual %d",
+			callsite, appHeight, height,
+		))
+	}
+	if appHeight <= app.cfg.BlobMaxBytesUpdateHeight {
+		return false
+	}
```

Comment:
This problematic. If `app.cfg.BlobMaxBytesUpdateHeight` is 0 and a non-0 blob size was set at genesis, we're returning `false` here, but there should be a non-0 blob.

Why do you need this "if", what breaks if you remove it?